src/main/webapp/trigger/help-CommitMessageContains_ja.html

```diff
@@ -0,0 +1,8 @@
+<p>
```

Comment:
This is not Japanese, but I wasn't sure what happens if this files does not exists, and the local is JA.  So I created it and included the English text.

Reply:
You can remove it. If there is no file for the current language then the default text, from the help file without a suffix, will be used.

Reply:
I will remove